repository_service_tuf/cli/admin/sign.py

```diff
@@ -24,13 +26,53 @@
     _filter_root_verification_results,
     _print_keys_for_signing,
     _print_root,
-    _warn_no_save,
 )
+from repository_service_tuf.helpers.api_client import (
+    URL,
+    Methods,
+    request_server,
+    send_payload,
+    task_status,
+)
+
+
+def _get_pending_roles(
+    settings: Any, api_server: Optional[str] = None
+) -> Dict[str, Dict[str, Any]]:
+    if api_server:
+        settings.SERVER = api_server
+
+    if settings.get("SERVER") is None:
+        api_server = Prompt.ask("\n[cyan]API[/] URL address")
+        settings.SERVER = api_server
+
+    response = request_server(
+        settings.SERVER, URL.METADATA_SIGN.value, Methods.GET
+    )
+    if response.status_code != 200:
+        raise click.ClickException(
+            f"Failed to retrieve metadata for signing. Error: {response.text}"
+        )
+
+    response_data: Dict[str, Any] = response.json().get("data")
+    if response_data is None:
+        raise click.ClickException(response.text)
+
+    pending_roles: Dict[str, Dict[str, Any]] = response_data.get(
+        "metadata", {}
+    )
+    if len(pending_roles) == 0:
+        raise click.ClickException("No metadata available for signing")
+
+    return pending_roles
 
 
 @metadata.command()  # type: ignore
-@click.argument("root_in", type=click.File("rb"))
-@click.argument("prev_root_in", type=click.File("rb"), required=False)
```

Comment:
**Note:** We can add it in a follow-up PR as well

I think it's still valid to keep it for local signing.
But I have some suggestions: 
1. Rename `root_in` for `SIGNING_JSON_INPUT`
   This input should be the JSON data from `GET [/api/v1/metadata/sign]` as a file input. The reason is that in the future, the user can sign any offline metadata (i.e., custom delegation targets)
2. Rename `prev_root_in` to `CURRENT_TRUSTED_METADATA`, to be any current trusted metadata`

It makes explicit and even more clear adding to the guide.